commit 06e88db43d1ff367d03b731c9c1f4f69da78149b
Author: Keno Fischer <kfischer@college.harvard.edu>
Date:   Fri Oct 9 17:24:54 2015 +0000

    Clear SectionSymbols in MCContext::Reset
    
    This was just forgotten when SectionSymbols was introduced and could cause
    corruption if the MCContext was reused after Reset.
    
    Reviewers: rafael
    
    Subscribers: llvm-commits
    
    Differential Revision: http://reviews.llvm.org/D13547
    
    git-svn-id: https://llvm.org/svn/llvm-project/llvm/trunk@249854 91177308-0d34-0410-b5e6-96231b3b80d8

diff --git a/lib/MC/MCContext.cpp b/lib/MC/MCContext.cpp
index ff195d0..2160b4e 100644
--- a/lib/MC/MCContext.cpp
+++ b/lib/MC/MCContext.cpp
@@ -80,6 +80,7 @@ void MCContext::reset() {
 
   UsedNames.clear();
   Symbols.clear();
+  SectionSymbols.clear();
   Allocator.Reset();
   Instances.clear();
   CompilationDir.clear();
